core/src/test/java/com/zextras/carbonio/files/TestUtils.java

```diff
@@ -46,7 +42,13 @@ public static Map<String, Object> jsonResponseToMap(String json, String operatio
         final Map<String, Object> data = (Map<String, Object>) result.get("data");
 
         if (data.get(operation) != null) {
-          return (Map<String, Object>) data.get(operation);
+          Object dataOperation = data.get(operation);
+          if(dataOperation instanceof ArrayList<?>) {
+            Map<String, Object> listResult = new HashMap<>();
+            listResult.put("listResult", dataOperation);
```

Comment:
In the GraphQL specs every successful result is returned in the `data` attribute
```suggestion
            listResult.put("data", dataOperation);
```